sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/MSIToken.java

```diff
@@ -79,8 +77,7 @@ private static Long parseDateToEpochSeconds(String dateTime) {
             System.err.println(e.getMessage());
         }
 
-        LOGGER.logAndThrow(new IllegalArgumentException(String.format("Unable to parse date time %s ", dateTime)));
-        return null;
+        throw new IllegalArgumentException("Unable to parse date time " + dateTime);
```

Comment:
Any reason not to use the logger here?

Reply:
It was using a static logger and we've generally been against using those. I can have this method spin up an instance of `ClientLogger` to log the exception.

Reply:
Created instance inside the method

Reply:
Also, check why `System.err` is used here. Maybe logger was not an option in this class? If logger can be used, we should use it instead of `System.err`.